src/Compilers/Core/Portable/PEWriter/Units.cs

```diff
@@ -56,167 +19,6 @@ internal interface IAssemblyReference : IModuleReference
         Version AssemblyVersionPattern { get; }
     }
 
-    /// <summary>
-    /// An object that represents a .NET module.
-    /// </summary>
-    internal interface IModule : IUnit, IModuleReference
```

Comment:
I forgot to move the comments to ```CommonPEModuleBuilder```. Will do, since most of the comments are actually useful.